lib/renderer/security-warnings.ts

```diff
@@ -73,7 +79,7 @@ const isUnsafeEvalEnabled = function () {
         return false
       }
       return true
-    }).toString()})()`, resolve)
+    }).toString()})()`, undefined, resolve)
```

Comment:
Is this a docs issue, or an issue in the code that typescript caught?

Reply:
The latter 🙈